ThirdPartyAdapters/unity/unity/src/main/java/com/google/ads/mediation/unity/UnityInitializer.java

```diff
@@ -0,0 +1,84 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.mediation.unity;
+
+import android.app.Activity;
+import android.util.Log;
+
+import com.unity3d.ads.BuildConfig;
+import com.unity3d.ads.IUnityAdsInitializationListener;
+import com.unity3d.ads.UnityAds;
+import com.unity3d.ads.metadata.MediationMetaData;
+
+/**
+ * The {@link UnityInitializer} is used to initialize Unity ads
+ */
+public class UnityInitializer {
+
+    /**
+     * UnityInitializer instance.
+     */
+    private static UnityInitializer unityInitializerInstance;
+
+    /**
+     * This method will return a
+     * {@link com.google.ads.mediation.unity.UnityInitializer} instance.
+     *
+     * @return the {@link #unityInitializerInstance}.
+     *
+     */
+    static UnityInitializer getInstance() {
+        if (unityInitializerInstance == null) {
+            unityInitializerInstance = new UnityInitializer();
+        }
+        return unityInitializerInstance;
+    }
+
+    /**
+     * This method will initialize {@link UnityAds}.
+     *
+     * @param activity    The Activity context.
+     * @param gameId      Unity Ads Game ID.
+     * @param initializationListener   Unity Ads Initialization listener.
+     *
+     */
+    public void initializeUnityAds(Activity activity, String gameId, IUnityAdsInitializationListener
```

Comment:
We don't require `Activity` to initialize SDK since 3.5. We can use `Context` from now.